specification/attributes/column_naming_convention.md

```diff
@@ -19,16 +18,20 @@ Naming convention for columns appearing in billing data.
 
 ## Requirements
 
-* Column IDs MUST use [Pascal case](https://techterms.com/definition/pascalcase).
-* Column IDs MUST NOT use abbreviations.
-* Column IDs SHOULD NOT use acronyms.
-* Column IDs MUST be alphanumeric with no special characters.
-* Columns that have an ID and a Name MUST have the Id or Name suffix in the Column ID. Display Name for a Column MAY
-  avoid the Name suffix if it is considered superfluous
+- All columns defined by FOCUS MUST follow the following rules:
+  - Column IDs MUST use [Pascal case](https://techterms.com/definition/pascalcase).
+  - Column IDs MUST NOT use abbreviations.
+  - Column IDs SHOULD NOT use acronyms.
+  - Column IDs MUST be alphanumeric with no special characters.
+  - Columns that have an ID and a Name MUST have the `Id` or `Name` suffix in the Column ID. Display Name for a Column MAY avoid the `Name` suffix if there are no other columns with the same name prefix.
+- Custom columns SHOULD follow the same rules as FOCUS columns listed above.
+- All custom column IDs MUST start with a vendor prefix string and an underscore (e.g., `abc_ColumnName`).
+  - Vendor prefixes MUST be lowercase, alphanumeric strings and SHOULD NOT be longer than 10 characters in length.
+  - Vendor prefixes MUST be representative of the company or brand that generated the column.
```

Comment:
There is a discussion about how to avoid prefix conflicts between different vendors and a proposition here:
https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/264/files

Another possibility to avoid conflicts: vendors could use URL domain names as vendor prefix, for example: azr.io, aws.io, etc ... Vendors might need to buy domain names to get short prefixes.

What's also interesting with this:
- As a practitioner,  when I discover a prefix, I can go to the website to discover which vendor it is
- A a vendor, I can document my FOCUS column on the website

Reply:
Can you leave this feedback on that PR? I'd like to separate registration from at least having a prefix.